server/routerlicious/packages/services-ordering-kafkanode/.eslintrc.cjs

```diff
@@ -10,10 +10,21 @@ module.exports = {
 	],
 	rules: {
 		"@typescript-eslint/strict-boolean-expressions": "off",
-		"import/no-nodejs-modules": "off",
+		// This package uses node's events APIs.
```

Comment:
This is a server-only package so I think we can revert all changes to this file and keep the previous version with no-nodejs-modules completely off.

Reply:
Let's remove the new comment, not necessary in a server package where it is ok to use nodejs modules